README.md

```diff
@@ -23,10 +23,21 @@ For more information, see:
 
 This project exists thanks to all the people who contribute and who have contributed in the past, whether as part of the long history of thousands of contributions to WordPress from many different people, or as contributions to ClassicPress itself.
 
-You can see a list of WordPress contributors by going to the [WordPress releases page](https://wordpress.org/news/category/releases/) and looking at the credits for each release.  All changes in WordPress 4.9.8 and earlier are part of ClassicPress, along with selected changes from later releases.
+You can help Fix bugs by submitting Pull Requests. Most of the bugs are shared via the [issues page](https://github.com/ClassicPress/ClassicPress/issues). Here are some pointers on how to:
 
-We are working on several ways to recognize people for their contributions to ClassicPress.  In the meantime, you can view a list of code contributions [here on GitHub](https://github.com/ClassicPress/ClassicPress/compare/LAST_WP_COMMIT...develop).
+- Fix Bugs: Generally when evaluating bugfix PRs, we look for these things (ideally all of them)
```

Comment:
This new content really needs to go into a `CONTRIBUTING.md` document that is linked from the readme, because I expect it to grow over time.